libs/sdk-ui-geo/api/sdk-ui-geo.api.md

```diff
@@ -36,7 +38,7 @@ export type CenterPositionChangedCallback = (center: IGeoLngLat) => void;
 export const CoreGeoChart: React_2.FC<ICoreGeoChartProps & WrappedComponentProps>;
 
 // @public (undocumented)
-export const GeoPushpinChart: React_2.ComponentType<Pick<IGeoPushpinChartProps, "locale" | "color" | "size" | "location" | "onError" | "config" | "onDrill" | "drillableItems" | "afterRender" | "onCenterPositionChanged" | "onZoomChanged" | "exportTitle" | "ErrorComponent" | "LoadingComponent" | "onExportReady" | "onLoadingChanged" | "pushData" | "backend" | "workspace" | "segmentBy" | "filters" | "sortBy">>;
+export const GeoPushpinChart: (props: IGeoPushpinChartProps) => JSX.Element;
```

Comment:
why this?

Reply:
Before:
`
const GeoPushpinChart = withTheme(withContexts(GeoPushpinChartInner));
`
Now:
`
const WrappedGeoPushpinChart = withTheme(withContexts(GeoPushpinChartInner));

/**
 * @public
 */
export const GeoPushpinChart = (props: IGeoPushpinChartProps) => {
    const [location, size, color, segmentBy, filters, sortBy] = useResolveValuesWithPlaceholders(
        [props.location, props.size, props.color, props.segmentBy, props.filters, props.sortBy],
        props.placeholdersResolutionContext,
    );

    return <WrappedGeoPushpinChart {...props} {...{ location, size, color, segmentBy, filters, sortBy }} />;
};
`